src/modelinstance.cpp

```diff
@@ -217,76 +344,39 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
-        shape_t shape(OVshape.begin(), OVshape.end());
-        auto mappingName = config.getMappingInputByKey(name);
-        auto tensorInfo = std::make_shared<TensorInfo>(name, mappingName, precision, shape);
-        std::string precision_str = tensorInfo->getPrecisionAsString();
-        this->inputsInfo[tensorInfo->getMappedName()] = std::move(tensorInfo);
-        std::stringstream shape_stream;
-        std::copy(shape.begin(), shape.end(), std::ostream_iterator<size_t>(shape_stream, " "));
-        SPDLOG_LOGGER_INFO(modelmanager_logger, "Input name: {}; mapping name: {}; shape: {}; effective shape; precision: {}; layout: ",
-            name, mappingName,
-            shape_stream.str(),
-            // effective_shape_stream.str()
-            precision_str);
-        //           TensorInfo::getStringFromLayout(input->getLayout()));
-    }
-    SPDLOG_INFO("Final network inputs_V1: {}", getNetworkInputsInfoString(networkInputs, config));
+    }
     return StatusCode::OK;
 }
 
 Status ModelInstance::loadOutputTensors(const ModelConfig& config) {
     this->outputsInfo.clear();
-    for (const auto& pair : network->getOutputsInfo()) {
-        const auto& name = pair.first;
-        auto output = pair.second;
-
-        // Data from network
-        auto precision = output->getPrecision();
-        auto layout = output->getLayout();
-
-        if (config.getLayouts().size() > 0) {
-            auto mappedName = config.getMappingOutputByKey(name);
-            auto it = config.getLayouts().find(mappedName == "" ? name : mappedName);
-            if (it != config.getLayouts().end()) {
-                layout = TensorInfo::getLayoutFromString(it->second);
-            }
-        }
 
-        output->setLayout(layout);
-
-        auto shape = output->getDims();
-        auto effectiveShape = output->getTensorDesc().getBlockingDesc().getBlockDims();
-        auto mappingName = config.getMappingOutputByKey(name);
-        auto tensor = std::make_shared<TensorInfo>(name, mappingName, precision, shape, layout);
-        std::string precision_str = tensor->getPrecisionAsString();
-        this->outputsInfo[tensor->getMappedName()] = std::move(tensor);
-        std::stringstream shape_stream;
-        std::copy(shape.begin(), shape.end(), std::ostream_iterator<size_t>(shape_stream, " "));
-        std::stringstream effective_shape_stream;
-        std::copy(effectiveShape.begin(), effectiveShape.end(), std::ostream_iterator<size_t>(effective_shape_stream, " "));
-        SPDLOG_INFO("Output name: {}; mapping name: {}; shape: {}; effective shape {}; precision: {}; layout: {}",
-            name, mappingName, shape_stream.str(), effective_shape_stream.str(), precision_str,
-            TensorInfo::getStringFromLayout(output->getLayout()));
-    }
-    auto outputTensors = this->network_2->outputs();
-    for (const auto& output : outputTensors) {
-        std::string name;
+    // TODO: Add layout postprocessing.
```

Comment:
Shouldn't this be done now as well>

Reply:
It is done in loadInputTensor. Needs to be done before reshape.